app/assets/javascripts/models/beta/iteration.js

```diff
@@ -30,13 +44,13 @@ export const getIterationForStory = (story, project) => {
 };
 
 const createSprint = (sprintNumber = 0, startDate = 0, isFiller = false, velocity) => ({
-  number: sprintNumber + 1,
+  number: sprintNumber,
   startDate: startDate,
   points: 0,
   completedPoints: null,
   stories: [],
   isFiller: isFiller,
```

Comment:
Same here about using the shorthand notation

```suggestion
  isFiller,
```